app/views/article/_research_starters.html.erb

```diff
@@ -0,0 +1,24 @@
+<% if params[:page].blank? || params[:page] == '1' -%>
+  <% research_starters = @response['research_starters'] -%>
+  <% if research_starters.present? -%>
+    <% starter = research_starters.first -%>
+    <div class='research-starter clearfix'>
+      <a class="research-starter-close" role="button" data-toggle="collapse" href="#research-starter-body" aria-expanded="true" aria-controls="research-starter-body">
+        <span id='research-starter-close-icon' class='fa fa-times-circle'></span>
+        <span class="sr-only">hide research starter</span>
+      </a>
+      <div id='research-starter-body' class='collapse.in'>
+        <h3>
+          <%= link_to starter['eds_title'], article_path(id: "#{starter['eds_database_id']}__#{starter['eds_accession_number']}") -%>
+        </h3>
+        <img src="<%= starter['eds_cover_thumb_url'] -%>"/>
```

Comment:
Are we guaranteed that the research starter will have a cover thumb?  Just want to make sure we don't unintentionally render a broken image here.